packages/perseus/src/mixins/provide-keypad.tsx

```diff
@@ -10,13 +10,14 @@
  * extend a `ProvideKeypad` component instead of using this mixin.
  */
 
-import {LegacyKeypad} from "@khanacademy/math-input";
+import {Keypad} from "@khanacademy/math-input";
```

Comment:
This is a nice outcome for this PR. As we begin removing the Legacy Keypad, perseus will be impacted less and less.